ext/standard/tests/url/parse_url_basic_011.phpt

```diff
@@ -12,19 +12,19 @@ var_dump(parse_url('127.0.0.1:9999#'));
 *** Testing parse_url() :can not recognize port without scheme ***
```

Comment:
I don't see the point in keeping this test. It has been introduced to show and verify that the other issue has been fixed; since we're going to revert, we should drop the test as well.

Reply:
:man_shrugging: Since in the original change no other test was touched that's a pretty clear indication that this behavior is untested. Keeping this would make it more obvious if it ever changes again. But I'm fine either way.

Reply:
Well, it shouldn't hurt to keep it, and it may help for future developement.

Reply:
I think it's ok to keep the comment as it documents current behavior